lib/containers/urls.pm

```diff
@@ -331,4 +331,16 @@ sub get_3rd_party_images {
     return (\@images);
 }
 
+# Get a list of container images under test
+# This list is either the list defined in CONTAINER_IMAGES_TO_TEST or the default container list for the currently running SLES version
+sub get_container_test_images {
+    my @images = get_urls_from_var('CONTAINER_IMAGES_TO_TEST');
+    if (@images) {
+        return @images;
+    } else {
```

Comment:
```suggestion
    unless ($image = get_urls_from_var('CONTAINER_IMAGES_TO_TEST')->[0]) {
        # Get list of images from CONTAINER_IMAGES_TO_TEST or use the default
```

Reply:
Done.

Reply:
Not yet :smile:

Reply:
You asked for [tests/containers/image.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14434/files#diff-8fcdf01dd9bd64ed5616c3f3673c309122ab4d99d4803d1161ea7d114d61d753) and that is done, no?